hpx/util/lightweight_test.hpp

```diff
@@ -47,9 +48,15 @@ struct fixture
 
   public:
     fixture(std::ostream& stream):
-      stream_(stream), sanity_failures_(0), test_failures_(0),
-      mutex_{BOOST_DETAIL_SPINLOCK_INIT}
+      stream_(stream), sanity_failures_(0), test_failures_(0)
+#if BOOST_VERSION >= 106200
+      , mutex_{BOOST_DETAIL_SPINLOCK_INIT}
```

Comment:
It doesn't. Buildbot failed for all gcc builders due to that (I fixed that on master now to have a clean build for non boost 1.62 builds).